x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -922,6 +936,27 @@ describe('When on the Trusted Apps Page', () => {
           backButtonUrl: '/fleet',
         });
       });
+
+      const priorMockImplementation = coreStart.http.get.getMockImplementation();
+      // @ts-ignore
```

Comment:
why the ts-ignore? is it due to the callback overload signatures? If so, I have gotten around this by just accepting `options` as the argument and doing a type assertion in the body of the function. see `x-pack/plugins/security_solution/public/common/mock/endpoint/http_handler_mock_factory.ts:199` for reference

Reply:
Changing this by using the getLastLoadedResourceState solves that

Reply:
Sorry I don't remember why I said that. This was copy&pasted from line 766.